types/react-native/index.d.ts

```diff
@@ -4039,6 +4029,21 @@ export class FlatList<ItemT> extends React.Component<FlatListProps<ItemT>> {
      * Displays the scroll indicators momentarily.
      */
     flashScrollIndicators: () => void;
+
+    /**
+     * Provides a handle to the underlying scroll responder.
+     */
+    getScrollResponder: () => JSX.Element;
```

Comment:
[Looks like](https://github.com/facebook/react-native/blob/41f4efb364ffa5ad8de6baf1fb8bb2470894eb28/Libraries/Lists/FlatList.js#L367) this should also be a Maybe:

```suggestion
    getScrollResponder: () => JSX.Element | null | undefined;
```

Reply:
Thanks! I overlooked this 😆